<launch>
    <arg name="fps" default="5" />
    <arg name="model" default="$(find neuro_car)/models/attention.model" />

    <node name="svm_attention" pkg="neuro_car" type="svm" output="screen">
        <param name="fps" type="int" value="$(arg fps)" />
        <param name="model" type="string" value="$(arg model)" />
    </node>
</launch>
